pkg/provider/apis/provider_spec.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+const (
+	// APIKey is a constant for a key name that is part of the Packet cloud credentials
+	APIKey          string = "apiToken"
+	AlternateAPIKey string = "alternateApiToken"
+	// V1alpha1 is the API version
+	V1alpha1 = "mcm.gardener.cloud/v1alpha1"
```

Comment:
We may have to mention the dataType for `v1alpha1` as string or the build fails

Reply:
That is odd, since go explicitly allows type inference. It doesn't hurt, though, so I don't mind adding it.

Reply:
Updated.